components/src/main/res/xml/scene_editing_mode.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Nice! I think this is the first opportunity we've had to use MotionScene in our app! :D